src/main/java/org/spongepowered/api/service/context/Context.java

```diff
@@ -74,20 +89,31 @@ public String getType() {
      * @return the specific name of the item involved in this context, for
      *         example if the type were {@code world} this would be the name of the
      *         world.
-     * 
      */
     public String getName() {
         return getValue();
     }
 
+    /**
+     * Get the weight.
+     *
+     * This value should be used to calculate the total weight of a Context set. A value of
```

Comment:
The second paragraph should be encapsuled in `<p>`.

Also it should be documented what this weight actually does. Are higher values "better" or lower values.

Last but not least, this method should be below the `getValue()` method.